src/components/DateRangePicker.jsx

```diff
@@ -228,9 +228,9 @@ export default class DateRangePicker extends React.Component {
 
   doesNotMeetMinimumNights(day) {
     const { startDate, focusedInput, minimumNights } = this.props;
-    const today = moment().startOf('day');
-    const dayDiff = startDate ? day.diff(startDate, 'days') : day.diff(today, 'days');
+    if (!startDate) return false;
 
+    const dayDiff = day.diff(startDate, 'days');
```

Comment:
`doesNotMeetMinimumNights = this.isBlocked(moment(day).subtract(minimumNights, 'days'));`

@majapw did you mean `doesNotMeetMinimumNights = this.props.isOutsideRange(moment(day).subtract(minimumNights, 'days'));`? Because `moment(day).subtract(minimumNights, 'days')` can be blocked for other reasons, in those case it is possible to select `day` as end date and some other non-blocked day before `moment(day).subtract(minimumNights, 'days')` as start date.